README.md

```diff
@@ -1,79 +1,99 @@
-# Tera - An Internet-Scale Database
+# Tera - A distributed high performance NoSQL database
 
 [![Build Status](http://220.181.7.231/buildStatus/icon?job=tera_master_build)](http://220.181.7.231/job/tera_master_build/)  
-Copyright 2015, Baidu, Inc.
+Copyright 2016, Baidu, Inc.
 
-Tera is a structured distributed database which is designed to manage trillions of links and webpages. Besides Webpage Database, Tera is used by many other Baidu products, including Baidu Trace System, Baidu User Behavior Analysis System, etc. Tera provides high performance and high scalability by taking fully advantage of new generation hardware.
+Tera is a distributed high performance NoSql database, which is inspired by google's [BigTable](http://static.googleusercontent.com/media/research.google.com/zh-CN//archive/bigtable-osdi06.pdf) and can manage __petabytes__ of data across thousands of commodity servers, and also it is widely used in many Baidu products, including web indexing, WebPage DB, LinkBase DB, etc. 
 
-# Features
-* Automatic and configurable sharding
+## Features
+
+* Linear and modular scalability
+* Automatic and configurable sharding of tables
+* MVCC
+* Column-oriented database and support locality group (like BigTable)
 * Ranged and hashed sharding strategies
-* Strong consistency
+* Strictly consistent reads and writes
+* Automatic failover support between tabletservers
 * Online schema change
-* Snapshot
-* Multi-level cache and Bloom Filters to suport real-time queries
+* Support snapshot on table
+* Support RAMDISK/SSD/DFS tiered storage
+* Block cache and Bloom Filters for real-time queries
+* Multi-type table support (RAMDISK/SSD/DISK table)
+* Easy to use [C++](doc/sdk_dev_guide.md)/[Java](sdk_dev_guide_for_java.md)/[python](sdk_dev_guide_for_python.md)/[REST-ful](doc/http_proxy.md) API for client access
+
+## Data Module
 
-# Contact Us
-{dist-lab, tera_dev, opensearch} at baidu.com
+Tera is a sparse, distributed, persistent multidimensional NoSql database. Table is indexed by a row key, column key, and a timestamp; each value in the table is an uninterpreted array of bytes.
```

Comment:
What we talk about when we talk about `indexed`?
It seems that what is the map is indexed, but not the table...